### DISCUSSION TITLE
LTSP and Snap

### DISCUSSION THREAD

**Discussion by JB-Prof** [ltsp/ltsp#497] [Q&A]
Hello,

I am running LTSP on Ubuntu Mate 18.04 LTS with no chroot and it works with no problems.  As per the installation documents when I did this, I removed Snap.

Our users now need to install a program that is only available via Snap.  Can I reinstall snap in order to install this application, or will this cause a problem with LTSP's function?

**Comment by alkisg** [comment]
Snap had various issues with overlayfs (netboot, live CDs), network mounted /home and tmpfs RAM usage in the past, but I think they were resolved; go ahead and try it.
